plugin/advisor/config/sql-review.yaml

```diff
@@ -8,7 +8,7 @@ categoryList:
   - COLUMN
   - DATABASE
 templateList:
-  - id: mysql.prod
+  - id: bb.sql-review.mysql.prod
```

Comment:
We should consider putting each template in a separate YAML file. The file won't contain categoryList since that can be defined in the code.